driver/flambda_backend_args.ml

```diff
@@ -27,6 +27,12 @@ let mk_dcfg f =
   "-dcfg", Arg.Unit f, " (undocumented)"
 ;;
 
+let mk_reorder_blocks_random f =
+  "-reorder-blocks-random",
```

Comment:
```suggestion
  "-dreorder-blocks-random",
```
or remove the leading "d" in "flambda_backend_flags.ml".

Reply:
I think that "-d" is prefix for "dump" debug, so I'll remove it from the comment.